static/app/components/metrics/mriSelect/index.tsx

```diff
@@ -65,7 +65,9 @@ export function getMetricsWithDuplicateNames(metrics: MetricMeta[]): Set<MRI> {
   const duplicateNames: string[] = [];
 
   for (const metric of metrics) {
-    const metricName = parseMRI(metric.mri)?.name;
+    const parsedMri = parseMRI(metric.mri);
+    // Include the use case to avaoid warning of conflicts between different use cases
+    const metricName = `${parsedMri?.useCase}_${parsedMri?.name}`;
```

Comment:
`metricName` will be always defined because of `_`

Reply:
that is actually fine as the we should always valid MRI's from the BE 👍 
The types of `parseMRI` are a bit lacking...

Reply:
@priscilawebdev  Follow up: https://github.com/getsentry/sentry/pull/73277

Reply:
but the code will never go to the `continue` described bellow . shall we remove it?

Reply:
that is true, good catch!
I would merge the other PR first and then remove the `continue` + change all `parsedMri?.` to `parsedMri.` in this file.

Reply:
done!